Userland/Libraries/LibCore/System.cpp

```diff
@@ -556,4 +548,17 @@ ErrorOr<void> utime(StringView path, Optional<struct utimbuf> maybe_buf)
 #endif
 }
 
+ErrorOr<Array<int, 2>> pipe2([[maybe_unused]] int flags)
+{
+    Array<int, 2> fds;
+#if defined(__serenity__) || defined(__linux__)
```

Comment:
I think all the *BSD's have `pipe2()` as well.

Reply:
:thinking: I guess I'll slap `__unix__` in there and expect everyone to cooperate, looks like only macos doesn't have `pipe2`, which conveniently doesn't define `__unix__` either.